Libraries/Microsoft.Teams.Apps/App.cs

```diff
@@ -52,11 +52,14 @@ internal string UserAgent
         }
     }
 
-    public App(AppOptions? options = null)
+    internal App() : this(null!, null)
+    { }
+
+    public App(IHttpCredentials credentials, AppOptions? options = null)
```

Comment:
we need to register our credentials in the DI, so we can get receive the IAuthorizationHeaderProvider